common/php-functions-test.cpp

```diff
@@ -0,0 +1,59 @@
+// Compiler for PHP (aka KPHP)
```

Comment:
Can we move this file somewhere to `tests/cpp`?

Reply:
Done.